march_hardware_interface/src/march_hardware_interface_node.cpp

```diff
@@ -21,10 +21,13 @@ int main(int argc, char** argv)
   AllowedRobot selected_robot = AllowedRobot(argv[1]);
   ROS_INFO_STREAM("Selected robot: " << selected_robot);
 
+  bool reset_imc = ros::param::param<bool>("~reset_imc", 1);
```

Comment:
Is this 1 the default value? Since this is a `bool` it would be better to use `true`, which is the same as 1. Actually, I think this default value should be `false`, since most of the time you don't want to reset and it agrees with the launch file.

Reply:
Indeed, thought i had already changed this. Seems I was not thinking clearly during this PR. wont happen again